src/molecules/navbar/navbar.tsx

```diff
@@ -74,7 +74,22 @@ export const NavbarBase = ({ config, theme }: NavbarTypes) => {
 };
 
 /**
- * Navbar Component uses the PublisherLogoHeader, HamburgerMenu and Hamburger atomic components internally.
+ * 
+ * Navbar Component by default uses the PublisherLogoHeader, HamburgerMenu and Hamburger atomic components internally.
+ * We can also render the customized navbar by passing the render prop navbarRender from the frontend inside app > server > app.js file under ampRoutes as shown below:
+ *
+ * ```javascript
+ * import React from "react";
+ * const CustomNavbar = ({ config, theme }) => (
+ *   <h1> CUSTOM NAVBAR </h1>
```

Comment:
it should be inside layout and head. refer footer docs . the link i shared with you